andela/technites-bn-frontend
# #169865680 Super Admin add Hosts

#### What does this PR do?

- This PR allows Super Admin to add Hosts

- This PR allows Host to reset his/her password

#### Description of Task to be completed?
Super Admin should be able to add a Host by providing his informations
#### How should this be manually tested?

1. Head to `addHost`

2. Use a Super Admin account.

3. Check the email received and use it to reset password on `host/reset`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169865680](https://www.pivotaltracker.com/story/show/169865680)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
